hathor/p2p/sync_v1/agent.py

```diff
@@ -218,7 +218,7 @@ def __init__(self, protocol: 'HathorProtocol', downloader: Downloader, reactor:
         # This number may decrease if a new transaction/block arrives in a timestamp smaller than it.
         self.synced_timestamp: int = 0
 
-        self.send_data_queue: SendDataPush = SendDataPush(self)
+        self.send_data_queue: Optional[SendDataPush] = None
```

Comment:
Where are we setting this?

Reply:
This is basically disabling the `send_data_queue`. I think we could just remove it.

Reply:
Made it separate on #892